controllers/index.js

```diff
@@ -47,6 +48,17 @@ exports.getCommentsByArticleId = (req, res, next) => {
     })
     .catch(next) 
 }
+exports.postCommentByArticleId = (req, res, next) => {
+    const {params, body} = req
+    if (typeof body.username !== 'string' && typeof body.body !== 'string') {
```

Comment:
lets allow psql to throw an error if its going to, this would probably have been picked up for breaking the rules of the columns.